keyboards/b_sides/rev41lp/keymaps/namnlos/README.md

```diff
@@ -0,0 +1,3 @@
+# Namnlos custom keymap for Rev41lp
```

Comment:
Please rename this file to lowercase.

Reply:
Will do when I'm back home! Am checked in at hospital so don't have any computer access at the moment.